force-app/main/default/lwc/recentSessions/recentSessions.js

```diff
@@ -143,7 +145,31 @@ export default class RecentSessions extends LightningElement {
     connectedCallback() {
         loadStyle(this, pmmFolder + "/recentSessionsOverrides.css");
         loadStyle(this, pmmFolder + "/listViewSelectorOverrides.css");
+        this.handleComponentSize();
+    }
+
+    renderedCallback() {
+        if (this.flexipageRegionWidth) {
+            return;
+        }
+
+        let sessionsComponent = this.template.querySelector(".sessions");
+
+        if (sessionsComponent) {
+            let sessionsComponentWidth = sessionsComponent.getBoundingClientRect().width;
+
+            if (sessionsComponentWidth <= this.SMALL_REGION_WIDTH) {
+                this.flexipageRegionWidth = "SMALL";
```

Comment:
Aligned with Jenny's comment above, would these strings perhaps be better as constants?